.github/workflows/pr-description.yml

```
@@ -15,6 +15,16 @@ on:
 permissions: {}
 
 jobs:
+  auto-assign:
+    name: Auto-assign author of a PR
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    if: github.event.action == 'opened'
+    permissions:
+      pull-requests: write
+    steps:
+      - uses: kentaro-m/auto-assign-action@v2.0.0
```

We should always reference a specific commit hash for untrusted action sources, because tags can be moved.

```suggestion
      - uses: kentaro-m/auto-assign-action@f4648c0a9fdb753479e9e75fc251f507ce17bb7e
```